site/content/docs/0.0/customize/options.md

```diff
@@ -15,6 +15,7 @@ You can find and customize these variables for key global options in OUDS Web's
 | Variable                          | Values                                | Description                                                                            |
 | --------------------------------- | ------------------------------------- | -------------------------------------------------------------------------------------- |
 | `$enable-bootstrap-compatibility` | `true` or `false` (default)           | Enforces Bootstrap compatibility. |
+| `$enable-css-grid`                | `true` or `false` (default)           | Enables the experimental CSS Grid system (e.g. `.grid`, `.g-col-md-1`, etc.). |
```

Comment:
@julien-deramond  Should be $enable-cssgrid, isn't it?

Reply:
Yep, https://github.com/twbs/bootstrap/pull/40818 is not merged yet upstream.